web-ng/src/app/components/card-view-project/card-view-project.component.scss

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@import '_variables';
+
+.project-card {
+  background-image: url(/assets/img/happy-land.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 160px;
+  height: 120px;
+  margin: 32px;
+  padding: 32px;
+}
+.project-add {
+  background-color: white;
```

Comment:
Please avoid color literals here and throughout; they will break dark mode, and are hard to maintain. Please use Material themes instead as we do elsewhere.

Reply:
updated.